cg/services/run_devices/pacbio/post_processing_service.py

```diff
@@ -49,11 +51,19 @@ def __init__(
         to_raise=PostProcessingError,
     )
     def post_process(self, run_name: str, dry_run: bool = False) -> None:
-        LOG.info(f"Starting PacBio post-processing for run: {run_name}")
+        LOG.info(f"Starting Pacbio post-processing for run: {run_name}")
         run_data: PacBioRunData = self.run_data_generator.get_run_data(
             run_name=run_name, sequencing_dir=self.sequencing_dir
         )
+        if self._is_run_post_processed(run_name):
```

Comment:
I would not have this check here, I would have it where we fetch the runs to post process.

Reply:
Here I disagree, as the purpose of the name service is to use it in other services, it is not exclusively a step in the post-processing